index.js

```diff
@@ -11,7 +11,12 @@ var SwaggerToHar = require('./swagger-to-har.js')
 var HTTPSnippet = require('httpsnippet')
 
 var getEndpointSnippets = function (swagger, path, method, targets) {
-  var har = SwaggerToHar.getEndpoint(swagger, path, method)
+  return getEndpointSnippetsWithQueryValues(swagger, path, method, targets, {})
```

Comment:
@annieying Is there a specific reason why we don't just make the last parameter optional, but keep the method name the same?